app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
```

Comment:
- [ ] I would recommend following the same design that's been given to you. Currently your `sign in` and `registration` page is not looking the same with the given wireframe.

### Given wireframe

![Screenshot from 2022-01-05 01-57-13](https://user-images.githubusercontent.com/22434660/148120738-7b4889d4-54a9-4442-8c84-bcff53e363d5.png)

### YOur design 

 - Sign up
 
 
![Screenshot from 2022-01-05 01-57-34](https://user-images.githubusercontent.com/22434660/148120691-0b530f1f-a28d-4d7d-85d7-99d4864c4401.png)

 
 - Sign in 
 
![Screenshot from 2022-01-05 01-58-11](https://user-images.githubusercontent.com/22434660/148120723-0e03e721-acde-4107-9a3b-76b614a769ee.png)

Reply:
Yes, now the styles are more like the given wireframe, thank you @Dipeshtwis for this recommendation 👍